can-bus/json/Debug/Debug_enum.json

```
@@ -0,0 +1,3 @@
+{
+    
+}
```

nit: missing blank line
```suggestion
}

```